core-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -119,12 +129,19 @@ public synchronized void stopBackgroundListening() {
     return states;
   }
 
-  private static void queryServerForBreakpoints(CloudDebugProcessState state, Debugger client) throws IOException {
```

Comment:
Looks like it now. I think there was a genuine change in this class at some point, but it went away during code review.